app/models/preprint.ts

```diff
@@ -100,6 +100,10 @@ export default class PreprintModel extends OsfModel {
         return this.dateWithdrawn !== null;
     }
 
+    get userHasAdminPermission(): boolean{
```

Comment:
might want to leave a `TODO:` comment here or actually implement this to check the `currentUserPermissions` array